stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/util/extensions/Fragment.kt

```diff
@@ -0,0 +1,16 @@
+package io.getstream.chat.ui.sample.util.extensions
+
+import android.view.WindowManager.LayoutParams.SOFT_INPUT_ADJUST_NOTHING
+import android.view.WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN
+import android.view.WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE
+import androidx.fragment.app.Fragment
+
+fun Fragment.useAdjustNothing() = setSoftInputMode(SOFT_INPUT_ADJUST_NOTHING)
+
+fun Fragment.useAdjustPan() = setSoftInputMode(SOFT_INPUT_ADJUST_PAN)
+
+fun Fragment.useAdjustResize() = setSoftInputMode(SOFT_INPUT_ADJUST_RESIZE)
+
+private fun Fragment.setSoftInputMode(flags: Int) {
+    (activity ?: return).window.setSoftInputMode(flags)
```

Comment:
Maybe activity?.window?.setSoftInputMode(flags)?

Reply:
It's the same thing, this just has fewer `?` in it 😄